workflow/rules/ref.smk

```diff
@@ -101,6 +101,7 @@ rule get_vep_cache:
 rule get_vep_plugins:
     output:
         directory("resources/vep/plugins"),
+        "resources/vep/plugins/LoFtool_scores.txt",
```

Comment:
This is always part of the VEP plugins. But we need to add this to the input of `annotate_variants` for correct handling of the `LoFtool`-plugin.